### PR TITLE
fix typos, copyright, k2/K2

### PR DESCRIPTION
**Pull request by iredwards**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -37,18 +37,18 @@ var logSuccess bool
 // progress spinner
 var terminalSpinner = spinner.New(spinner.CharSets[35], 200*time.Millisecond)
 
-// init the k2 config viper instance
+// init the K2 config viper instance
 var clusterConfig = viper.New()
 
-// init the k2cli config viper instance
+// init the K2cli config viper instance
```
Comment (joejulian) [inline cmd/root.go]: Let's keep this one `k2cli`

File: cmd/root.go [hunk 2]
```diff
@@ -77,7 +77,7 @@ func init() {
 		"k2config",
 		"k",
 		"",
-		"config file for k2cli (default \""+os.Getenv("HOME")+"/.k2cli.yaml)\"")
+		"config file for k2cli (default \""+os.Getenv("HOME")+"/.k2cli.yaml\")"
```
Comment (joejulian) [inline cmd/root.go]: It was correct the first time. The `\"` is the quote character within the string. The `"` ends the string that began on this line. The `)` closes the function that started on line 53.

Comment (joejulian) [inline cmd/root.go]: Ah, I see what the intention is. It should be
```
"config file for k2cli (default \""+os.Getenv("HOME")+"/.k2cli.yaml\")")
```

File: cmd/root.go (hunk 1 above)
Comment (iredwards) [inline cmd/root.go]: yes.  my bad.

**Comment by joejulian** [comment]
LGTM

**Pull request merged**